HSDP-1D.md

```diff
@@ -0,0 +1,89 @@
+```
```

Comment:
Is there some reason this entire spec is a code block in markdown?  Makes it miss out on the formatting that could be happening.